build/version.go

```diff
@@ -25,7 +39,12 @@ func BuildTypeString() string {
 	}
 }
 
-const BuildVersion = "1.22.0"
+// Intent: Major.Network.EvenReleases
+var BuildVersionArray = [3]int{1, 22, 0}
```

Comment:
What's the advantage of this array?

Reply:
elsewhere (make_debs.go) interpret the version. Having a standard of it being numbers makes that interpretation simpler.